domain/src/main/java/org/oppia/android/domain/topic/StoryProgressTestHelper.kt

```diff
@@ -100,7 +120,127 @@ class StoryProgressTestHelper @Inject constructor(
+   *
+   * @param profileId The profile we are setting topic progress for.
+   *
+   * @param timestampOlderThanOneWeek If the timestamp for completing the topic is from more than one week ago.
+   */
+  fun markFullTopicProgressForTestTopics(profileId: ProfileId, timestampOlderThanOneWeek: Boolean) {
+    val timestamp = if (!timestampOlderThanOneWeek) {
+      getCurrentTimestamp()
+    } else {
+      getOldTimestamp()
+    }
+    // Stories and Explorations for "Test Topic"s are not in chronological order so we want to ensure
+    // that the combinations of Topic / Story / Exploration that are visible will be marked as completed.
+    storyProgressController.recordCompletedChapter(
+      profileId,
+      TEST_TOPIC_ID_0,
+      TEST_STORY_ID_0,
+      TEST_EXPLORATION_ID_2,
+      timestamp
+    )
+    storyProgressController.recordCompletedChapter(
+      profileId,
+      TEST_TOPIC_ID_0,
+      TEST_STORY_ID_0,
+      TEST_EXPLORATION_ID_5,
+      timestamp
+    )
+    storyProgressController.recordCompletedChapter(
+      profileId,
+      TEST_TOPIC_ID_0,
+      TEST_STORY_ID_1,
+      TEST_EXPLORATION_ID_3,
+      timestamp
+    )
+    storyProgressController.recordCompletedChapter(
+      profileId,
+      TEST_TOPIC_ID_0,
+      TEST_STORY_ID_1,
+      TEST_EXPLORATION_ID_4,
+      timestamp
+    )
+    storyProgressController.recordCompletedChapter(
+      profileId,
+      TEST_TOPIC_ID_1,
+      TEST_STORY_ID_2,
+      TEST_EXPLORATION_ID_4,
+      timestamp
+    )
+    storyProgressController.recordCompletedChapter(
+      profileId,
+      TEST_TOPIC_ID_1,
+      TEST_STORY_ID_2,
+      TEST_EXPLORATION_ID_5,
+      timestamp
+    )
+  }
+
+  /**
+   * Marks full topic progress on Ratios for a particular profile.
+   *
+   * @param profileId The profile we are setting topic progress for.
+   *
+   * @param timestampOlderThanOneWeek If the timestamp for completing the topic is from more than one week ago.
+   */
+  fun markFullTopicProgressForRatios(profileId: ProfileId, timestampOlderThanOneWeek: Boolean) {
+    val timestamp = if (!timestampOlderThanOneWeek) {
+      getCurrentTimestamp()
+    } else {
+      getOldTimestamp()
+    }
+    storyProgressController.recordCompletedChapter(
+      profileId,
+      RATIOS_TOPIC_ID,
+      RATIOS_STORY_ID_0,
+      RATIOS_EXPLORATION_ID_0,
+      timestamp
+    )
+    storyProgressController.recordCompletedChapter(
+      profileId,
+      RATIOS_TOPIC_ID,
+      RATIOS_STORY_ID_0,
+      RATIOS_EXPLORATION_ID_1,
+      timestamp
+    )
+    storyProgressController.recordCompletedChapter(
+      profileId,
+      RATIOS_TOPIC_ID,
+      RATIOS_STORY_ID_1,
+      RATIOS_EXPLORATION_ID_2,
+      timestamp
+    )
+    storyProgressController.recordCompletedChapter(
+      profileId,
+      RATIOS_TOPIC_ID,
+      RATIOS_STORY_ID_1,
+      RATIOS_EXPLORATION_ID_3,
+      timestamp
+    )
+  }
+
+  /** Marks one story progress full in ratios exploration for a particular profile.
```

Comment:
Please fix KDoc formatting here & below (missing newline after ``/**`` and have extra newline between ``@param``s).

Reply:
Done